src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,23 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+public class FindCommand extends Command {
+    private String query;
+
+    public FindCommand(String query) {
+        this.query = query;
+    }
+    @Override
```

Comment:
```suggestion
    }
    
    @Override
```

Whitespace here too! :)